bin/web/views/index.ejs

```diff
@@ -360,27 +399,77 @@ ${fields}
+                let currentCategory = "";
+                let currentOptgroup = null;
                 prompts.forEach(prompt => {
                     const option = document.createElement('option');
                     option.value = prompt.value;
                     option.textContent = prompt.name;
-                    promptSelect.appendChild(option);
+
+                    const category = getCategoryName(prompt.name);
+
+                    if (category !== currentCategory) {
+                        currentCategory = category;
+                        currentOptgroup = promptSelect.querySelector(`optgroup[label="${category}"]`);
+                        if (!currentOptgroup) {
+                            currentOptgroup = document.createElement('optgroup');
+                            currentOptgroup.label = category;
+                            promptSelect.appendChild(currentOptgroup);
+                        }
+                    }
+
+                    if (!currentOptgroup) {
+                        currentOptgroup = document.createElement('optgroup');
+                        currentOptgroup.label = "other";
+                        promptSelect.appendChild(currentOptgroup);
+                    }
+
+                    const subOption = document.createElement('option');
+                    subOption.value = prompt.value;
+                    subOption.textContent = getPromptName(prompt.name);
+                    currentOptgroup.appendChild(subOption);
                 });
 
+                function getCategoryName(name) {
+                    const categoryIndex = name.indexOf('-');
+                    if (categoryIndex !== -1) {
+                        return name.slice(categoryIndex + 1);
+                    }
+                    return "other";
+                }
+
+                function getPromptName(name) {
+                    const categoryIndex = name.indexOf('-');
+                    if (categoryIndex !== -1) {
+                        return name.slice(0, categoryIndex);
+                    }
+                    return name;
+                }
+
                 // Select the "default" prompt if it exists, otherwise select the first prompt
                 const defaultPrompt = prompts.find(prompt => prompt.name.toLowerCase() === 'default');
                 const initialPrompt = defaultPrompt || prompts[0];
                 promptSelect.value = initialPrompt.value;
                 input.textContent = initialPrompt.value;
 
-                promptSelect.addEventListener('change', handlePromptChange);
+                promptSelect.addEventListener('change', updateInputToPrompt);
                 resetButton.addEventListener('click', (e) => {
                     e.preventDefault() // Prevent form from submitting
-                    handlePromptChange()
+                    updateInputToPrompt()
                 });
 
-
+                updateInputToPrompt();
```

Comment:
when you load the app for the first time it autofocuses the input and selects the >PROMPT section :)